### PR TITLE
add a command to output parsed config after env var substitution

### PR DESCRIPTION
**Pull request by mattbennett**
Useful for debugging services that have complicated config files with many environment variable substitutions.

### CODE REVIEW

File: nameko/cli/commands.py [hunk 1]
```diff
@@ -46,6 +46,31 @@ def main(args):
...
 
+class Config(Command):
+    """ Output as YAML string the configuration that would be passed to a
+    service.
+
+    Useful for viewing config files that load values from environement
+    variables.
+    """
+
+    name = 'config'
```
Comment (davidszotten) [inline nameko/cli/commands.py]: how about calling this `show-config` or similar to make it more clearly distinct from `--config`

Comment (mattbennett) [reply nameko/cli/commands.py]: That is a much better name!

**Comment by davidszotten** [comment]
nice idea! like it

File: nameko/cli/config.py [hunk 2]
```diff
@@ -1,11 +0,0 @@
-from __future__ import print_function
```
Comment (davidszotten) [inline nameko/cli/config.py]: old file removed but new file not added?

Comment (mattbennett) [reply nameko/cli/config.py]: Whoops!

File: test/cli/test_config.py [hunk 3]
```diff
@@ -0,0 +1,41 @@
+from textwrap import dedent
```
Comment (davidszotten) [inline test/cli/test_config.py]: `test_show_config`?

File: test/cli/test_config.py [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    config = tmpdir.join('config.yaml')
+    config.write("""
+        FOO: ${FOO:foobar}
+        BAR: ${BAR}
+    """)
+
+    parser = setup_parser()
+    setup_yaml_parser()
+    args = parser.parse_args([
+        'config',
```
Comment (davidszotten) [inline test/cli/test_config.py]: `show-config`...

**Comment by mattbennett** [comment]
Sorry for the back and forth. All comments addressed now, and I've checked the test actually pass ;)

**Comment by mattbennett** [comment]
Thanks. Will add changelog entry when I cut a release

**Pull request merged**